Libraries/Text/TextInput/Multiline/RCTMultilineTextInputView.m

```diff
@@ -149,7 +149,8 @@ - (void)textInputDidChange
 
 - (void)setAttributedText:(NSAttributedString *)attributedText
 {
-  [_backedTextInputView setAttributedText:attributedText];
+  [super setAttributedText:attributedText];
```

Comment:
Calling super calls it on the backedTextInputView too?

Reply:
We need to call `setAttributedText` on the superclass `RCTBaseTextInputView`, **not** `NSTextView` which is the `_backedTextInputView`

https://github.com/microsoft/react-native-macos/blob/97ad878d8f9056ebd298e605d72448b9c7502e47/Libraries/Text/TextInput/RCTBaseTextInputView.m#L150